platform/pulp/tasking/tasks.py

```diff
@@ -225,14 +237,24 @@ def apply_async(self, *args, **kwargs):
 
         :param group_id:    The id that identifies which group of tasks a task belongs to
         :type group_id:     uuid.UUID
+        :param args:          The positional arguments to pass on to the task.
+        :type  args:          tuple
+        :param kwargs:        The keyword arguments to pass on to the task.
+        :type  kwargs:        dict
+        :param **options:     For all options accepted by apply_async please visit:
+                              http://docs.celeryproject.org/en/latest/reference/celery.app.task.html#celery.app.task.Task.apply_async  #NOQA
```

Comment:
Oh nevermind this is just github wrapping the line. You've already done what I asked for.